main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -464,7 +464,7 @@ object Resolver {
               mapN(defsVal, assocsVal) {
                 case (defs, assocs) =>
                   val traitUse = TraitSymUse(trt.sym, trt0.loc)
-                  ResolvedAst.Declaration.Instance(doc, ann, mod, traitUse, tpe, tconstrs, assocs, defs, Name.mkUnlocatedNName(ns), loc)
+                  ResolvedAst.Declaration.Instance(doc, ann, mod, traitUse, tpe, tconstrs.collect { case Some(t) => t }, assocs, defs, Name.mkUnlocatedNName(ns), loc)
```

Comment:
Instead of this inline collect, I would prefer that introduce a local variable for tconstrs.

Reply:
Oops yeah my bad